content/en/docs/tasks/administer-cluster/unknown-version-interoperability-proxy.md

```diff
@@ -0,0 +1,47 @@
+---
+reviewers:
+- jpbetz
+title: Unknown Version Interoperability Proxy Alpha
+content_type: concept
+weight: 220
+---
+
+<!-- overview -->
+This page provides an overview of a new capability added to kube-apiservers that enables them to proxy a resource request to the correct peer kube-apiserver when there are multiple apiservers at varied versions in a cluster.
+
+### Unknown Version Interoperability Proxy Alphs
+
+{{< feature-state state="alpha"  for_k8s_version="v1.28" >}}
```

Comment:
How about:
```suggestion
{{< feature-state state="alpha"  for_k8s_version="v1.28" >}}

Kubernetes {{< skew currentVersion >}} includes an alpha feature that lets one
{{< glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}}
proxy a resource request to a different _peer_ API server. This is useful when there are multiple
API servers running different versions of Kubernetes in one cluster (for example, during a long-lived
rollout to a new release of Kubernetes).

This mechanism is called the _Unknown Version Interoperability Proxy_.
```

Avoid mentioning that a feature is new, even when it is. We like our docs to be as [timeless](https://developers.google.com/style/timeless-documentation) as is reasonably practical.